### ISSUE TITLE
Don't use bind in WebPack2

### ISSUE THREAD

**Issue by arv** [webpack/webpack#2140]
WebPack2 seems to add `bind()` when calling imported functions. `bind` is really slow so this is a bad idea. A better solution is to do `(0, f)()` instead of `f.bind()()`.

**Comment by sokra** [comment]
yeah, I need to find a better solution here. Would be `f.call(null)` fine?

**Comment by arv** [comment]
`f.call(null)` is better than `bind` but still not as good as `(0,f)()`

**Comment by tgriesser** [comment]
Also this is an issue with lodash, which is a function which also has a bind property, and throws if the first argument to `_.bind()` is not a function when using the module w/ chaining syntax:

```js
import {someValue} from './module1'
import _ from 'lodash'

function someFn() {
  return _(someVal).map(someFn).value()
}
```

```js
// throws
return /* harmony import */__WEBPACK_IMPORTED_MODULE_1_lodash___default.a.bind()(someVal).map(someFn).value()
```

**Comment by mAAdhaTTah** [comment]
> (0,f)()

This is how `babel` does it for calling destructured functions.

**Comment by tgriesser** [comment]
Is this fixed? Still seeing this in 2.1.0-beta.15
